### ISSUE TITLE
Auto Compile for arm64

### ISSUE THREAD

**Issue by schooldanlp6** [hrkfdn/ncspot#1555]
**Is your feature request related to a problem? Please describe.**
I daily a linux phone for quite a while now. And I always need to clean build when I update ncspot. This is not a problem initself but when it breaks which happend like 2 times this year and I am on the go I need to type the bloody commands with my hand which is quite unpractical to say the least.

**Describe the solution you'd like**
 an arm64 build especially for a bit less compute power devices would be nice.

**Describe alternatives you've considered**
Else you could automate the things over action even if only for the arm build this would be helping me a lot

**Additional context**
(There is not really any)

**Comment by hrkfdn** [comment]
I had to remove cross compilation as the setup was rather clunky, maybe with https://github.com/github/roadmap/issues/970 implemented we can add this back again in Q1 2025.

**Comment by schooldanlp6** [comment]
Yes I understand that it does not make sense to cross compile when it is painful to setup (which is why I currently don't cross compile too). Well sounds nice tho

**Comment by hrkfdn** [comment]
Solved with #1581
